packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -352,6 +366,16 @@ const CreatePassword = ({ password, platform, validatePassword, onSubmit, error_
                                 )}
                             </PasswordMeter>
                         </div>
+                        {is_real_financial_stp && !has_mt5_account && !is_bvi && (
+                            <div className='dc-modal__container_cfd-password-modal__description'>
+                                <Localize i18n_default_text='Your MT5 Financial STP account will be opened through Deriv (FX) Ltd. All trading in this account is subject to the regulations and guidelines of the Labuan Financial Service Authority (LFSA). None of your other accounts, including your Deriv account, is subject to the regulations and guidelines of the Labuan Financial Service Authority (LFSA).' />
+                            </div>
+                        )}
+                        {is_real_financial_stp && !has_mt5_account && is_bvi && (
```

Comment:
Deriv FX needs to cater for both bvi and non-bvi accounts. please update the code.